firefox_ui_harness/decorators.py

```diff
@@ -10,8 +10,11 @@
 
 def skip_if_e10s(target):
     def wrapper(self, *args, **kwargs):
-        if self.appinfo.browserTabsRemoteAutostart:
-            raise SkipTest("Skipping due to e10s")
+        try:
+            if self.appinfo.browserTabsRemoteAutostart:
```

Comment:
Can I ask what kind of Firefox build are you using for testing your patch? Is it the latest Nightly build? Getting this property out of the appinfo object should not throw an exception but return false.